### DISCUSSION TITLE
Google authentication

### DISCUSSION THREAD

**Discussion by Nyctophilist** [community/community#50433] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I want to implement a new feature in my project that will allow users to log in using their Google accounts. Currently, the project only supports email and password authentication, but we want to add the option for users to log in with their Google credentials for convenience. Any ideas for how i can achieve this?

I have made a little research, but I wanted to know if anyone knew an easy way to implement it.

**Comment by Berkanktk** [comment, marked as answer]
I would take a look at [Firebase](https://firebase.google.com/) from Google. This can also give you the ability to login using Facebook and Twitter. I don't know the size of the project, but they have a free tier (spark plan) that should be fine and easy to implement.
